src/main/java/com/ansa/simple/StringUtils.java

```diff
@@ -8,12 +8,42 @@
 public class StringUtils {
 
     public static int countSymbolInString(String input, char ch) {
+        if (input==null){
+            throw new IllegalArgumentException();
+
+        }
+        int count=0;
+        for (int index=0;index<input.length();index++){
+            if (input.charAt(index)==ch){
+                count++;
+            }
+        }
+
         //implementation here
-        return 0;
+        return count;
     }
 
     public static Map<Character, Integer> countSymbols(String input){
-        return new HashMap<>();
+        Map<Character,Integer> charCounter=new HashMap<Character,Integer>();
+        int count=0;
+        char[]char_array =input.toCharArray();
```

Comment:
why to use intermediate variable char_array? Naming notation is not for Java - charArray if the variable is required. But are you sure you need it?